Created the Radio Button Component

yashmouje opened the pull request.

giancarlopernudisegura reviewed (changes requested):
Need to modify the `index.tsx` file to import and re-export this new component. Also if possible, can you create an example in the `examples` dir?

yashmouje commented:
> Need to modify the `index.tsx` file to import and re-export this new component. Also if possible, can you create an example in the `examples` dir?

I have imported and re-exported the component in `index.tsx`. I have created an example called `radio.tsx` in the `examples` dir. I have also added a `styles` prop to the `RadioButton.tsx` component.

giancarlopernudisegura reviewed (changes requested):
Good job on the disabled prop. Let's go ahead and rename the component filename to just Radio.tsx and move it to the input folder. Also, some padding might be good as the label is too close to the radio button. Finally, can we change the default browser defined blue color to the primary color of the natUI theme? This last part might be easiest by adding a style rule in the base [natUI repo](https://github.com/neuralbertatech/natUI/). Let me know if you'd need help with that.

giancarlopernudisegura commented on examples/pages/radio.tsx:
```diff
@@ -0,0 +1,29 @@
+      }}>
+        
+        <Radio text="Option 1" name="radio-test" style={{padding: "10px"}}></Radio>
+        <Radio text="Option 2" name="radio-test" style={{padding: "10px"}}></Radio>
```
Another thing, change this `text` prop to a children prop (the example from the #10).
https://beta.reactjs.org/learn/passing-props-to-a-component#passing-jsx-as-children

yashmouje commented:
`components/inputs/Radio.tsx` [React Component]
  - Moved to `inputs` dir
  - Replaced `text` Prop with `children` Prop

`src/index.tsx`
  - Updated Imports and Exports
  
`components/inputs/index.tsx`
  - Updated Imports and Exports

`examples/pages/radio.tsx`
  - Removed Style Property. Added styles for Radio Button in natUI. Will create new PR for that.

giancarlopernudisegura reviewed (approved):
Looks good. I ended up simplifying the component. Couple of things:
1. Removed the ternary operator on the disabled prop and just passed the prop value to the disabled parameter.
2. Added a unique ID (prefixed with the name). This is how the radio+labels work together in HTML instead of nesting the input tag inside the label tag.
3. Inlined the common props.
4. Added the disbaled param to the jsdocs.
